templates/solochain/pallets/template/src/benchmarking.rs

```diff
@@ -0,0 +1,35 @@
+//! Benchmarking setup for pallet-template
```

Comment:
Header missing?

Reply:
These files didn't have a license header beforehand, not adding them here to minimize diff, but it is a follow-up.

Reply:
Maybe worth adding something here for the tempaltes: https://github.com/paritytech/polkadot-sdk/blob/81156d03cec0b34c62faa853c3b38e3ba7c82ced/.github/workflows/check-licenses.yml#L40